.vsts-dotnet.yml

```
@@ -34,7 +34,7 @@ variables:
     - name: SourceBranch
       value: ''
   - name: EnableReleaseOneLocBuild
-    value: true
+    value: false # Do NOT merge this line to vs17.11
```

I was thinking something like 

```suggestion
    value: false # Disable loc for vs17.10 now that it's shipped
```

and where it's the current branch name, and update it everywhere, then it'll conflict when we hit

```diff
- # Disable loc for vs17.10 now that it's shipped
+ # Enable loc for vs17.11
```

I think?

---

We will need to update the version in this comment during the creating the new release branch to make it conflict later. But we can a comment have in main:
# Enable loc for main
where during the final branding PR or later in other PR we update just a comment for future conflict.

---

I will write it in our release instruction